proto/cosmos/upgrade/v1beta1/query.proto

```diff
@@ -64,5 +64,5 @@ message QueryUpgradedConsensusStateRequest {
 // QueryUpgradedConsensusStateResponse is the response type for the Query/UpgradedConsensusState
 // RPC method.
 message QueryUpgradedConsensusStateResponse {
-  google.protobuf.Any upgraded_consensus_state = 1;
+  bytes upgraded_consensus_state = 1;
```

Comment:
This is proto-breaking. Could we just add a new field instead? Or else we can also bump to v1beta2

Reply:
I can do whichever you prefer. The old fields would just be unused if I added a new field. It would also be unusable since I cannot fulfill the `UnpackInterfaces` function since there won't be a interface to unpack into. I presume I'd have to change this for the Upgrade Plan as well or does only the Query definitions matter?

Reply:
Not bumping to v1beta2 would be ideal. If the old field is not used anymore, I would say let's go for `reserved`, like [here](https://github.com/cosmos/cosmos-sdk/pull/7802#discussion_r562714165)

See https://developers.google.com/protocol-buffers/docs/proto#updating

> Non-required fields can be removed, as long as the field number is not used again in your updated message type. You may want to rename the field instead, perhaps adding the prefix "OBSOLETE_", or make the field number reserved, so that future users of your .proto can't accidentally reuse the number.

Reply:
sweet, thanks for catching this. Updated it, let me know if anything is wrong

Reply:
The proto-breakage check still says breaking, but I presume this is fine?

Do I need the `reserved "option";` part?